src/main/java/com/netgrif/application/engine/workflow/domain/Case.java

```diff
@@ -107,7 +111,6 @@ public class Case {
 
     @Getter
     @Setter
-    @JsonIgnore
```

Comment:
why was this annotation removed?

Reply:
TaskPair object that consists of task id and transition id mapping is now used on the frontend.